docs/en/reference/configuration.md

```diff
@@ -379,6 +379,10 @@ Applications may also need to declare the permissions they require. Permissions
 
 Briefcase maintains a set of cross-platform permissions:
 
+#### `permission.bluetooth`
+
+Permission to connect to an external device via Bluetooth.
```

Comment:
This is all that is needed for configuration; however, there is also a need to document the specific manifestation of permissions in the platform guide for android/gradle. 

There's two relevant sections:
1. The `permissions` attribute should clarify how dictionary values are interpreted
2. The "Permissions" section should detail the default mapping of the Bluetooth permission on Android.